paginator.js

```diff
@@ -759,7 +756,7 @@ export class Paginator extends HTMLElement {
         }))
         this.#scrollToAnchor(this.#anchor)
     }
-    get scrolled() {
```

Comment:
This is a public API for the `Paginator`. It's not proper to refactor in this way without unit tests. And there is not even compiled time type checking for this since it's pure javascript. I suggest we don't do this right now.

Reply:
Oh okay, no worries - will remove the commit for this one